packages/core/.storybook/manager.jsx

```diff
@@ -24,6 +24,13 @@ addons.setConfig({
       }
 
       return <SidebarItem status={parameters.status}>{name.replace(storyStatus, "").trim()}</SidebarItem>;
+    },
+    filters: {
+      patterns: filterInternalStoryDocsPage
     }
   }
 });
+
+function filterInternalStoryDocsPage(item) {
+  return process.env.STORYBOOK_ENV === "development" || !item.tags?.includes?.("internal");
```

Comment:
Storybook uses env variables, I think you can get them somehow here

Reply:
I've tried to also use their support, and their docs, couldn't find anything related

Reply:
would love to know if you find something, I'll change it 🙏🏼

Reply:
You can use `process.env.NODE_ENV`

Reply:
Are they setting it?

Reply:
Yes tried it locally also, and they use `production` for build

Reply:
what do they supply in `dev`?
I'm not sure if we're exporting as production build